### ISSUE TITLE
Padded sequences?

### ISSUE THREAD

**Issue by drozzy** [DiffEqML/torchdyn#57]
Just curious if this works with padded sequences?
https://pytorch.org/docs/stable/generated/torch.nn.utils.rnn.pad_sequence.html

I have sequences of variable length and I would like to use torchdyn for classification. 
Currently I am doing windowing, but I would rather use padding.

**Comment by Zymrael** [comment]
Hi @drozzy, could you provide a more specific example of what you're trying to do? There are several models in `torchdyn`, developed for different tasks. Although we have not designed any component with padded seqs in mind, it might be easy to use them with some of these architectures, depending on your objective.

**Comment by drozzy** [comment]
Maybe my question is wrong. I'm basically working with variable length sequences and I want to do classification using torchdyn.


From torchdyn I'm just using something like:
model = NeuralDE(f, sensitivity='adjoint', solver='dopri5').to(device)

**Comment by drozzy** [comment]
I am not sure what I was asking. I'll close it and re-open it if I figure it out!
